From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@slaven-s__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

slaven-s commented:
Thank you very much!Slaven
 


     Datuma 21:45 Utorak, 1. Rujan 2015., Azure Contribution License Agreements <notifications@github.com> je napisao:
   

 Hi @slaven-s, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
 
 This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
 TTYL, AZPRBOT;—
Reply to this email directly or view it on GitHub.

rmca14 commented:
@JennieHubbard 
Please review and approve/disapprove, thanks.
@barbkess 
FYI

JennieHubbard commented:
#sign-off

Commit:
task 343: Remove note stating advanced access required for Request Header Regex, miscl acro edits

maestersid committed:
Rework of Android guides to use FCM instead of GCM.

maestersid committed:
Grammar Updates

maestersid committed:
Corrected spelling errors and small edits

maestersid committed:
final grammar edit

Commit:
adding a note for free/shared tiers

maestersid committed:
Removing note

Commit:
Update app-service-dev-test-note.md

CawaMS committed:
Update vs-azure-tools-storage-explorer-relnotes.md

correct latest version

spelluru committed:
fixed issues

spelluru committed:
one more fix

AdiGanMSFT committed:
Update backup-mabs-protection-matrix.md

ktoliver committed:
edit pass: notification-hubs-push-notification-fixer.md

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into notification-hubs-push-notification-fixer

ktoliver committed:
more edits

ktoliver committed:
Merge pull request #30550 from mmacy/aci-consistency-refresh

 [aci] links, arg ordering, and refresh

v-geberr committed:
Include all predicted intent scores

GitHubber17 committed:
Merge pull request #30637 from CawaMS/master

Update vs-azure-tools-storage-explorer-relnotes.md

Lisaco88 committed:
Merge pull request #30639 from spelluru/adfssisv21228

fixed issues

Lisaco88 committed:
Merge pull request #30634 from dksimpson/dean-343

task 343: Remove note stating advanced access required for Request He…

Lisaco88 committed:
Merge pull request #30629 from tfitzmac/1228ps2

removed include file

v-geberr committed:
add pub link

Lisaco88 committed:
Merge pull request #30628 from tfitzmac/1228psfix

add secure password

Ja-Dunn committed:
Merge pull request #30604 from tdykstra/precompiled

remove "precompiled", add Cosmos DB warnings

ktoliver committed:
Merge pull request #30593 from jwhiteDev/xamarin-droid-getstarted-update

Xamarin.Android Getting Started update

Ja-Dunn committed:
Merge pull request #30538 from ShawnJackson/how-to-scale-assessment2

edit pass: how-to-scale-assessment

kgremban committed:
Fix broken links to IoT developer center

Commit:
Changes and new image.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aaslogging

Commit:
Update ToC.

Commit:
Fix bad link.

v-geberr committed:
enter not tab

ShawnJackson committed:
Merge branch 'master' into cdn-rules-engine-reference-match-conditions

mdshihab committed:
(ForumsTeam) Minor Grammar Edit

rayne-wiselman committed:
Update tutorial-prepare-on-premises-hyper-v.md

rayne-wiselman committed:
Update tutorial-hyper-v-to-azure-failover-failback.md

rayne-wiselman committed:
Update tutorial-dr-drill-azure.md

rayne-wiselman committed:
Update tutorial-prepare-azure.md

rayne-wiselman committed:
Update tutorial-hyper-v-to-azure-failover-failback.md

rayne-wiselman committed:
Update tutorial-hyper-v-to-azure.md

rayne-wiselman committed:
Update tutorial-hyper-v-vmm-to-azure.md

rayne-wiselman committed:
Update tutorial-prepare-on-premises-hyper-v.md

rayne-wiselman committed:
Update support-matrix-hyper-v-to-azure.md

rayne-wiselman committed:
adding new article to TOC

rayne-wiselman committed:
Update support-matrix-hyper-v-to-azure.md

rayne-wiselman committed:
Update support-matrix-hyper-v-to-azure.md

rayne-wiselman committed:
Update support-matrix-hyper-v-to-azure.md

rayne-wiselman committed:
Update tutorial-hyper-v-to-azure-failover-failback.md

rayne-wiselman committed:
Update tutorial-hyper-v-to-azure-failover-failback.md

gsacavdm committed:
Improve formatting for terms & concepts

gsacavdm committed:
Improve formatting vm-common-log-collector-ext

gsacavdm committed:
Give logs location code formatting

gsacavdm committed:
Fix some of the Acrolinx issues.

gsacavdm committed:
Fix code formatting

gsacavdm committed:
More minor formatting fixes

gsacavdm committed:
Fixing a bunch of Acrolinx issues

PRMerger14 committed:
Merge pull request #29700 from mdshihab/patch-9

(ForumsTeam) Grammar edit

Commit:
Merge pull request #29278 from mdshihab/patch-8

(ForumsTeam) Typo

PRMerger19 committed:
Merge pull request #29178 from mdshihab/patch-7

(ForumsTeam) Added a space character

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

v-nagta committed:
USER STORY 1152922 SaaS App Tutorial: Citrix GotoMeeting update - III

Sogup committed:
Update azure-backup-limits.md

zhouwangzw committed:
Use "Next steps" 

Update to use "Next steps" according to feedback from the DocTeam.

zhouwangzw committed:
Add reference to get-started articles in TOC

Add reference to get started articles in TOC, based on feedback from the DocTeam

zhouwangzw committed:
Update toc.yml

Fix errors

Sogup committed:
Update backup-azure-arm-vms-prepare.md

zhouwangzw committed:
Update toc.yml

fix typo

rayne-wiselman committed:
Update toc.yml

dominicbetts committed:
Tidy the title

dominicbetts committed:
Fix command casing

dominicbetts committed:
Add embedded video

YuriDio committed:
updating articles

ktoliver committed:
edit pass: relay-hybrid-connections-dotnet-get-started.md

PRMerger10 committed:
Merge pull request #30065 from ronortloff/master

Updated firewall content

georgewallace committed:
automation : fixing link

ktoliver committed:
more edits

Ja-Dunn committed:
Merge pull request #30672 from georgewallace/patch-2

automation : fixing link

GitHubber17 committed:
Merge pull request #30560 from msjuergent/opsguide

wave1

Ja-Dunn committed:
Merge pull request #30670 from YuriDio/UpdatesJan2018

updating articles

GitHubber17 committed:
Merge pull request #30599 from rayne-wiselman/rayne-december26

fixing customer issue in article

Ja-Dunn committed:
Merge pull request #30665 from dominicbetts/fix-casing

Fix command casing

Ja-Dunn committed:
Merge pull request #30664 from dominicbetts/tidy-title

Tidy the title

Ja-Dunn committed:
Merge pull request #30662 from Sogup/patch-20

Update backup-azure-arm-vms-prepare.md

Ja-Dunn committed:
Merge pull request #30661 from zhouwangzw/patch-5

Add reference to get-started articles in TOC

Ja-Dunn committed:
Merge pull request #30659 from zhouwangzw/patch-4

Use "Next steps"

Ja-Dunn committed:
Merge pull request #30658 from Sogup/patch-19

Update azure-backup-limits.md

v-geberr committed:
call out enter as UX step

Ja-Dunn committed:
Merge pull request #30653 from gsacavdm/patch-5

Improve formatting vm-common-log-collector-ext

Ja-Dunn committed:
Merge pull request #30652 from gsacavdm/patch-4

Improve formatting for terms & concepts

Ja-Dunn committed:
Merge pull request #30414 from ShawnJackson/cdn-rules-engine-reference-match-conditions

edit pass: cdn-rules-engine-reference-match-conditions

Ja-Dunn committed:
Merge pull request #30673 from v-geberr/0102-howto-phraselist-enter

call out enter as UX step

GitHubber17 committed:
Merge pull request #30645 from v-geberr/1229-phraselist-synons

Include all predicted intent scores

Ja-Dunn committed:
Merge pull request #30646 from kgremban/dec29_linkfix

Fix broken links to IoT developer center

dominicbetts committed:
Fix embed syntax

mimig1 committed:
Incorporated feedback from ealsur.

v-geberr committed:
new subdomain for austrailia

ktoliver committed:
more edits

CarlRabeler committed:
Update sql-database-get-started-portal.md

CarlRabeler committed:
Update sql-database-manage-logins.md

Ja-Dunn committed:
Merge pull request #30651 from mdshihab/patch-11

(ForumsTeam) Minor Typo Correction

Commit:
Merge pull request #30271 from carolinacmoravia/patch-514

M16849: Possible incorrect term:  "Manage Keys Access > Manage access…

gsacavdm committed:
One more fix for extensions-diagnostics-template

mimig1 committed:
acrolinx fixes

ktoliver committed:
fix line spacing

cynthn committed:
Update virtual-machines-common-sizes-storage.md

CarlRabeler committed:
Update role-based-access-built-in-roles.md

cynthn committed:
Merge pull request #6 from cynthn/cynthn-patch-7

Update virtual-machines-common-sizes-storage.md

Ja-Dunn committed:
Merge pull request #30677 from CarlRabeler/patch-370

Update sql-database-get-started-portal.md

Ja-Dunn committed:
Merge pull request #30678 from CarlRabeler/patch-371

Update sql-database-manage-logins.md

CarlRabeler committed:
Update sql-database-firewall-configure.md

mimig1 committed:
incorporated feedback from Rajesh

tdykstra committed:
change doc db api to Azure Cosmos DB

mimig1 committed:
Merge branch 'master' into 12072017javaperf

mimig1 committed:
corrected date

mimig1 committed:
updated date

ShawnJackson committed:
Merge pull request #30547 from ktoliver/indexing-policies

edit pass: indexing-policies.md

rayne-wiselman committed:
Update toc.yml

Ja-Dunn committed:
Merge pull request #30654 from gsacavdm/patch-6

Fix code formatting

Ja-Dunn committed:
Merge pull request #30681 from CarlRabeler/patch-372

Update role-based-access-built-in-roles.md

mimig1 committed:
Cosmos DB: Graph GA updates

rayne-wiselman committed:
Update toc.yml

Ja-Dunn committed:
Merge pull request #30685 from tdykstra/patch-1

change doc db api to Azure Cosmos DB

iainfoulds committed:
Removing H2 on SKUs. This is dynamically available based on underlying capacity

mimig1 committed:
fixed broken link

tynevi committed:
Update .acrolinx-config.edn

JasonWHowell committed:
Update concepts-servers.md

Lisaco88 committed:
Merge pull request #30360 from mimig1/12192017

Cosmos DB: Updated firewall doc for portal changes

Lisaco88 committed:
Merge pull request #30686 from iainfoulds/azskus

Availability Zone VM SKUs

seanli1988 committed:
Merge branch 'b2' of https://github.com/seanli1988/azure-docs-pr into b2

ktoliver committed:
Update notification-hubs-push-notification-fixer.md

ktoliver committed:
Update notification-hubs-push-notification-fixer.md

JasonWHowell committed:
Update concepts-servers.md

rmca14 committed:
Merge pull request #30351 from rayne-wiselman/rayne-december19-3

updating hyper-v tutorials

sanjeev3 committed:
Update quick-start.md

Lisaco88 committed:
Merge pull request #30427 from rnagpal/patch-27

Connection String update

HJToland3 committed:
online migration topic addition

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

rmca14 committed:
Merge pull request #30687 from JasonWHowell/patch-3

PostgreSQL: Update concepts-servers.md

sethmanheim committed:
Fix broken link, and freshness updates

Lisaco88 committed:
Merge pull request #30689 from seanli1988/b2

added known issues

barbkess committed:
concurrency slot fix

HJToland3 committed:
online migration article tweaks

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

Commit:
fixed article based on comment

Commit:
automation: updates

Commit:
automation: updates

Lisaco88 committed:
Merge pull request #30692 from sethmanheim/sb-fix

Fix broken link, and freshness updates

rmca14 committed:
Merge pull request #29648 from mimig1/12072017javaperf

Cosmos DB: New Java performance tips article

rmca14 committed:
Merge pull request #30694 from barbkess/performance

concurrency slot fix

mimig1 committed:
Cosmos DB: Updated min value for partitioned coll.

rmca14 committed:
Merge pull request #30698 from msebolt/java-tpm-fix

fixed article based on comment

gunjanj committed:
Update agent failure for default website

@wmgries @jeffplat @Klaasl, @tamran
Minor troubleshooting tip for default website name failure

Lisaco88 committed:
Merge pull request #30683 from CarlRabeler/patch-373

Update sql-database-firewall-configure.md

Commit:
updating the remaining articles

rayne-wiselman committed:
Update toc.yml

Lisaco88 committed:
Merge pull request #30695 from HJToland3/master

Online migration topic additions

rmca14 committed:
Merge pull request #30610 from rayne-wiselman/rayne-december27

adding hyper-v to azure support matrix

ChrisGMsft committed:
Updated per Kelly's feedback

HJToland3 committed:
online migration article typo fix

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

Commit:
Fixed markdown error in table.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aaslogging

HJToland3 committed:
online migration tweak

ShawnJackson committed:
Merge pull request #30644 from ktoliver/notification-hubs-push-notification-fixer

edit pass: notification-hubs-push-notification-fixer.md

ChrisGMsft committed:
Some grammar improvements

Commit:
Removed PBIE reference

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aaslogging

ChrisGMsft committed:
More style fixes

rmca14 committed:
Merge pull request #30636 from SyntaxC4-MSFT/cfowler-free-shared-tier

Adding a note to clarify the use of Free/Shared Tier

cynthn committed:
Update virtual-machines-windows-powershell-upload-generalized-script.md

cynthn committed:
Merge pull request #7 from cynthn/cynthn-patch-8

Update virtual-machines-windows-powershell-upload-generalized-script.md

Juliako committed:
updated conneciting with REST topic

Commit:
Minor grammar corrections.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aaslogging

alkohli committed:
modified TOC to remove SVA MPIO

JasonWHowell committed:
Update quickstart-create-server-database-azure-cli.md

JasonWHowell committed:
Update quickstart-create-server-database-azure-cli.md

Ja-Dunn committed:
Merge pull request #30701 from mimig1/01022018minrus

Cosmos DB: Updated min value for partitioned coll.

JasonWHowell committed:
Update quickstart-create-server-database-azure-cli.md

Ja-Dunn committed:
Merge pull request #30676 from v-geberr/0102-subdomains

new subdomain for austrailia

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update credentials.md

GitHubber17 committed:
Merge pull request #30319 from ChrisGMsft/patch-15

Update with additional AMQP details

sanjeev3 committed:
Update toc.yml

sanjeev3 committed:
Update credentials.md

msjuergent committed:
firstchange

cynthn committed:
Update virtual-machines-windows-powershell-upload-generalized-script.md

mimig1 committed:
acrolinx fixes

Ja-Dunn committed:
Merge pull request #30711 from Juliako/AMSRESTUpdates

updated conneciting with REST topic

alkohli committed:
deleted and added a redirect to SVA MPIO article

Ja-Dunn committed:
Merge pull request #30705 from HJToland3/master

Online migration article tweak

sanjeev3 committed:
Update credentials.md

alkohli committed:
fixed the source URL

sanjeev3 committed:
Update credentials.md

GitHubber17 committed:
Merge pull request #30647 from Minewiskan/aaslogging

Aaslogging

sanjeev3 committed:
Update credentials.md

Ja-Dunn committed:
Merge pull request #30696 from mimig1/01022018graphga

Cosmos DB: Graph GA updates

sanjeev3 committed:
Update quick-start.md

sanjeev3 committed:
Update quick-start.md

Ja-Dunn committed:
Merge pull request #30682 from cynthn/master

Ls IOPS]\Throughput fixes

Ja-Dunn committed:
Merge pull request #30710 from JasonWHowell/patch-3

PostgreSQL: Update quickstart-create-server-database-azure-cli.md

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into relay-hybrid-connections-dotnet-get-started

Ja-Dunn committed:
Merge pull request #30709 from alkohli/SVAMPIO

modified TOC to remove SVA MPIO

Ja-Dunn committed:
Merge pull request #30713 from sanjeev3/master

Updated Quick Start and Credentials Articles.

ktoliver committed:
change formatting of text code references

HJToland3 committed:
online migration article updates

HJToland3 committed:
tweak

msjuergent committed:
secondchange

Ja-Dunn committed:
Merge pull request #30718 from HJToland3/master

minor tweak to online migration

gunjanj committed:
Update storage-sync-files-troubleshoot.md

cynthn committed:
Update change-drive-letter.md

Commit:
Freshness - connect to a file share

Commit:
acrolinx

Lisaco88 committed:
Update storage-sync-files-troubleshoot.md

Lisaco88 committed:
Merge pull request #30702 from gunjanj/patch-5

Update agent failure for default website

Lisaco88 committed:
Merge pull request #30722 from cynthn/patch-51

Update change-drive-letter.md

Lisaco88 committed:
Merge pull request #30725 from cynthn/file-share

Freshness - connect to a file share

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

dominicbetts committed:
Fix embed syntax

dominicbetts committed:
Make sure video query returns content

dominicbetts committed:
Update for stretch Raspbian

dominicbetts committed:
Tidy formatting

dominicbetts committed:
Fix layout

Brenduns committed:
Fixing instance of blog for blob, plus minor edit for FAQ

daveirwin1 committed:
Updated SLES kernels and removed sunsetted SLES and Oracle versions

Commit:
updates to article

itechedit committed:
Merge pull request #30624 from Sogup/patch-18

Added an error cause and solution

Commit:
CLF fix

Commit:
corrected spacing

tdykstra committed:
fix link

Brenduns committed:
Sweep for acrolynk driven fixes

itechedit committed:
Merge pull request #30642 from AdiGanMsft/patch-6

Update backup-mabs-protection-matrix, Add Azure Stack

rmca14 committed:
Merge pull request #30667 from dominicbetts/suite-video

Add embedded video

itechedit committed:
Merge pull request #30699 from georgewallace/autoupdates0102

automation : updates

rmca14 committed:
Merge pull request #30714 from msjuergent/hlioverviewchange

firstchange

itechedit committed:
Merge pull request #30734 from dominicbetts/fix-video-links

Make sure video query returns content

ktoliver committed:
change ms.date year to 2018

Change year to 2018

itechedit committed:
Merge pull request #30737 from daveirwin1/master

Updated SLES kernels and removed sunsetted SLES and Oracle versions

rmca14 committed:
Merge pull request #30735 from dominicbetts/update-c-raspberry

Update for stretch Raspbian

ktoliver committed:
Merge pull request #30740 from tdykstra/patch-2

fix link

itechedit committed:
Update azure-stack-usage-related-faq.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rmca14 committed:
Merge pull request #30739 from raymondlaghaeian/patch-21

CLF fix

ktoliver committed:
Merge pull request #30736 from Brenduns/daily-0103

Fixing instance of blog for blob, plus minor edit for FAQ

ktoliver committed:
Merge pull request #30657 from v-nagta/citrix

USER STORY 1152922 SaaS App Tutorial: Citrix GotoMeeting update - III

itechedit committed:
Merge pull request #30741 from Brenduns/topic-sweep

Topic sweep

v-alje committed:
Merge pull request #30748 from MicrosoftDocs/FromPublicRepo

From public repo

ShawnJackson committed:
Merge pull request #30671 from ktoliver/relay-hybrid-connections-dotnet-get-started

edit pass: relay-hybrid-connections-dotnet-get-started.md
